package.json

```diff
@@ -140,12 +144,13 @@
     "rxjs": "6.4.0",
     "ts-node": "^5.0.1",
     "utility-types": "^3.7.0",
-    "web3": "1.2.4",
-    "web3modal": "1.5.0"
+    "web3modal": "^1.1.0",
+    "webpack": "^4.36.1",
+    "ws": "^7.1.0"
   },
   "devDependencies": {
     "@babel/plugin-syntax-dynamic-import": "^7.7.4",
-    "@daostack/migration": "0.0.1-rc.41-v4",
+    "@dorgtech/test-env-experimental": "4.0.9",
```

Comment:
why test-env is imported to alchemy ?

Reply:
It's used inside the tests. It gets the test DAO addresses from the daos.json. See test/integration/utils.ts